src/protocols/secure_channel/CASESession.cpp

```diff
@@ -413,7 +391,8 @@ CHIP_ERROR CASESession::HandleSigmaR1(System::PacketBufferHandle & msg)
     TLV::TLVType containerType = TLV::kTLVType_Structure;
 
     uint16_t encryptionKeyId = 0;
-    uint32_t n_trusted_roots;
+    uint8_t destinationIdentifier[kMAX_Hash_Length];
```

Comment:
For this to be future-safe, we need to use `kSHA256_Hash_Length` rather than kMAX_Hash_Length everywhere, since destinationId is organized based on SHA256

Reply:
Updated.